diff --git a/src/anbox/graphics/emugl/RenderApi.cpp b/src/anbox/graphics/emugl/RenderApi.cpp
index 9ca2b47..9417102 100644
--- a/src/anbox/graphics/emugl/RenderApi.cpp
+++ b/src/anbox/graphics/emugl/RenderApi.cpp
@@ -30,6 +30,7 @@ GLESv1Dispatch s_gles1;
 
 namespace {
 constexpr const char *default_egl_lib{"libEGL.so.1"};
+constexpr const char *default_glesv1_lib{"libGLES_CM.so.1"};
 constexpr const char *default_glesv2_lib{"libGLESv2.so.2"};
 }
 
@@ -37,6 +38,7 @@ namespace anbox::graphics::emugl {
 std::vector<GLLibrary> default_gl_libraries() {
   return std::vector<GLLibrary>{
     {GLLibrary::Type::EGL, default_egl_lib},
+    {GLLibrary::Type::GLESv1, default_glesv1_lib},
     {GLLibrary::Type::GLESv2, default_glesv2_lib},
   };
 }
